### PR TITLE
Feat(eos_cli_config_gen): add cvconfig flag to TerminAttr

### PR DESCRIPTION
**Pull request by perimore**
## Change Summary

Add cvconfig flag to daemon TerminAttr config

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Simply add the option for `cvconfig` under `daemon_terminattr` config.

## How to test
Tested with:

- `cvconfig: true` results in `exec /usr/bin/TerminAttr ... -cvconfig`
- `cvconfig: false` results in `exec /usr/bin/TerminAttr ...`
- omitting `cvconfig:` results in `exec /usr/bin/TerminAttr ...`

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2 [hunk 1]
```diff
@@ -89,6 +89,9 @@ daemon TerminAttr
 {%     endif %}
 {%     if daemon_terminattr.sflowaddr is arista.avd.defined %}
 {%         set cvp_config.cli = cvp_config.cli ~ " -sflowaddr=" ~ daemon_terminattr.sflowaddr %}
+{%     endif %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2:92]: I am not sure, but I think these options were put in some order. @noredistribution should we keep these options in a certain order?

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md [hunk 2]
```diff
@@ -470,6 +470,7 @@ daemon_terminattr:
   ipfixaddr: <str>
   sflow: <bool>
   sflowaddr: <str>
+  cvconfig: <bool>
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md:474]: did you add this by hand? You should run the molecule `molecule converge -s build_schemas_and_docs` locally to auto generate everything from the fragment file.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2 (hunk 1 above)
Comment (noredistribution) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2:92]: no, the order doesn't matter
testing this now as well

**Comment by noredistribution** [comment]
can you please also update molecule scenarios, e.g.: https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/terminattr-extra-flags.yml

@ClausHolbechArista do we need to update it in the eos_cli_config_gen_v4.0 folder?

**Comment by ClausHolbechArista** [comment]
> can you please also update molecule scenarios, e.g.: https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/terminattr-extra-flags.yml
> 
> @ClausHolbechArista do we need to update it in the eos_cli_config_gen_v4.0 folder?

no need to update 4.0 molecule.

**Comment by perimore** [comment]
> 

I updated 4.0 molecule anyway, does it matter?

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2 (hunk 1 above)
Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2:92]: if possible, we like to match the order of what the CLI will return when doing a `show running-config`

Comment (noredistribution) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2:92]: yea that's true in general, but daemon configs are not ordered, they are as is as they are basically bash statements

**Review by ClausHolbechArista — APPROVED** [review]

**Review by noredistribution — APPROVED** [review]
LGTM! Tested again with normal and dual streaming config too and ran also molecule locally and generated fine

**Pull request merged**